layers/+readers/pdf/packages.el

```diff
@@ -108,7 +108,7 @@
         "gh"               'pdf-outline-up-heading
         "gg"               'beginning-of-buffer
         "G"                'pdf-outline-end-of-buffer
-        "TAB"              'outline-toggle-children
+        (kbd "<tab>")      'outline-toggle-children
```

Comment:
you can bind it to both (kbd "TAB") and (kbd "<tab>")

See https://www.emacswiki.org/emacs/TabKey

Reply:
Ah that's interesting, there is more information also [here](https://emacs.stackexchange.com/questions/17509/how-to-distinguish-c-i-from-tab).

Anyway, as pdf-tools can not be used in terminals, it makes not sense to bind `C-i` to `outline-toggle-children` so it seems to me that the current PR is 'correct',